### PR TITLE
Privileges check

### PR DESCRIPTION
**Pull request by sabahoth01**

### CODE REVIEW

File: src/app/AppRouter.tsx [hunk 1]
```diff
@@ -56,21 +56,50 @@ const routes: Record<AppRoutes, AppRouteProps> = {
   [AppRoutes.EVENT_LIST]: {
     path: RoutePaths.eventList,
     authenticated: true,
+    authorized: anyPrivilege(
+      new Set([
+        new PrivilegeData(PrivilegeNames.VIEW_ALL_EVENTS),
+        new PrivilegeData(PrivilegeNames.CREATE_EVENT)
```
Comment (anamrzv) [inline src/app/AppRouter.tsx:62]: add SEARCH_EVENTS_AND_ACTIVITIES

Comment (anamrzv) [reply src/app/AppRouter.tsx:62]: add EDIT_EVENT_INFO

File: src/app/AppRouter.tsx [hunk 2]
```diff
@@ -56,21 +56,50 @@ const routes: Record<AppRoutes, AppRouteProps> = {
...
+      ])
+    ),
   },
   [AppRoutes.EVENT_DATA]: {
     path: RoutePaths.eventData,
     authenticated: true,
+    authorized: anyPrivilege(
+      new Set([
+        new PrivilegeData(PrivilegeNames.VIEW_ALL_EVENTS),
```
Comment (anamrzv) [inline src/app/AppRouter.tsx:71]: add EDIT_EVENT_INFO

File: src/app/AppRouter.tsx [hunk 3]
```diff
@@ -56,21 +56,50 @@ const routes: Record<AppRoutes, AppRouteProps> = {
...
   },
   [AppRoutes.TASK_LIST]: {
     path: RoutePaths.taskList,
     authenticated: true,
+    authorized: anyPrivilege(
+      new Set([
+        new PrivilegeData(PrivilegeNames.VIEW_ALL_EVENT_TASKS),
+        new PrivilegeData(PrivilegeNames.DECLINE_TASK_EXECUTION),
+        new PrivilegeData(PrivilegeNames.ASSIGN_SELF_AS_TASK_EXECUTOR)
+      ])
```
Comment (anamrzv) [inline src/app/AppRouter.tsx:92]: i think here CHANGE_ASSIGNED_TASK_STATUS only
remove other

File: src/app/AppRouter.tsx [hunk 4]
```diff
@@ -56,21 +56,50 @@ const routes: Record<AppRoutes, AppRouteProps> = {
...
+        new PrivilegeData(PrivilegeNames.VIEW_EVENT_ACTIVITIES),
+        new PrivilegeData(PrivilegeNames.SEARCH_EVENTS_AND_ACTIVITIES),
+        new PrivilegeData(PrivilegeNames.VIEW_ORGANIZER_USERS),
+        new PrivilegeData(PrivilegeNames.VIEW_ALL_EVENT_TASKS),
+        new PrivilegeData(PrivilegeNames.CREATE_EVENT_ACTIVITIES),
+        new PrivilegeData(PrivilegeNames.EDIT_EVENT_INFO),
+        new PrivilegeData(PrivilegeNames.EDIT_EVENT_ACTIVITIES),
+        new PrivilegeData(PrivilegeNames.IMPORT_PARTICIPANT_LIST_XLSX),
+        new PrivilegeData(PrivilegeNames.EXPORT_PARTICIPANT_LIST_XLSX)
+      ])
```
Comment (anamrzv) [inline src/app/AppRouter.tsx:81]: keep only: VIEW_ALL_EVENTS, 
SEARCH_EVENTS_AND_ACTIVITIES
EDIT_EVENT_INFO

other privileges are checked in code of EventData page

File: src/app/AppRouter.tsx (hunk 1 above)
Comment (anamrzv) [reply src/app/AppRouter.tsx:62]: delete CREATE_EVENT
In this file we only check for VIEW privileges, other privileges are checked inside specific pages

**Comment by anamrzv** [comment]
@sabahoth01 please edit file

**Review by zubrailx — APPROVED** [review]

**Pull request merged**